### PR TITLE
Java files custom dir

### PR DESCRIPTION
**Pull request by acheshkov**

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#308](https://github.com/yegor256/aibolit/issues/308) is assigned to @lyriccoder[/z](https://www.0crat.com/u/lyriccoder), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/fbdabd51-933a-4931-b828-f3016423257a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: scripts/03-calculate-metrics.py [hunk 1]
```diff
@@ -2,10 +2,11 @@
 import pandas as pd
 import os
 from pathlib import Path
+from aibolit.config import Config
 
 
 DIR_TO_CREATE = 'target/03'
-dir_to_analyze = './target/01'
+dir_to_analyze = Config.java_files_folder() or './target/01'
```
Comment (lyriccoder) [inline scripts/03-calculate-metrics.py:9]: remove and add command line arguments, do not use env variable here

File: aibolit/ml_pipeline/ml_pipeline.py [hunk 2]
```diff
@@ -22,9 +20,14 @@ def collect_dataset(java_folder):
...
+    print('Directory with JAVA classes: ', java_folder)
     print('Filtering java files...')
-    result = subprocess.run(['make', 'filter'], stdout=subprocess.PIPE)
+
+    filter_cmd = ['make', 'filter']
+    if java_folder is not None:
+        filter_cmd.append(f'dir={java_folder}')
+
+    result = subprocess.run(filter_cmd, stdout=subprocess.PIPE)
     if result.returncode != 0:
```
Comment (lyriccoder) [inline aibolit/ml_pipeline/ml_pipeline.py:31]: add command line parameter to 03-metrics
```
print('Download PMD and compute metrics...')
    result = subprocess.run(['make', 'metrics', java_folder], stdout=subprocess.PIPE)
```

File: test/config/test_config.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# import os
+from unittest import TestCase
+# from aibolit.config import Config
+
+
+class TestConfig(TestCase):
```
Comment (lyriccoder) [inline test/config/test_config.py:6]: what is this?

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @lyriccoder[/z](https://www.0crat.com/u/lyriccoder)[/z](https://www.0crat.com/u/lyriccoder) didn't make CR comments

**Pull request merged**